.github/workflows/build-test-lint.yaml

```diff
@@ -9,4 +9,4 @@ on:
 
 jobs:
   call-workflow:
```

Comment:
This change was requested by LeaAnn/Team Sprint. They deleted a branch that we were pointing to. Not updating this causes the pipeline's build to fail.